.dockerignore

```
@@ -2,4 +2,6 @@
 **.env
 
 node_modules
-**/Dockerfile
\ No newline at end of file
+**/Dockerfile
+
+environments
```

```suggestion
/environments/
```

I'm not sure if this is correct notation to ignore only the top-most environments folder, but we should guard it in case this ignores unintended files or folders.